codalab/lib/bundle_cli.py

```diff
@@ -662,12 +662,34 @@ def print_table(
         """
         Pretty-print a list of columns from each row in the given list of dicts.
         """
-        # Get the contents of the table
         rows = [columns]
+        # display restricted fields if the server returns those fields - which suggests the user is root
+        try:
+            if row_dicts and row_dicts[0].get('last_login'):
+                columns += ('last_login', 'time', 'disk', 'parallel_run_quota')
+                rows = [columns]
+        except KeyError:
+            pass
+
+        # Get the contents of the table
         for row_dict in row_dicts:
             row = []
             for col in columns:
-                cell = row_dict.get(col)
+                try:
+                    if col == 'time':
```

Comment:
I don't know if we want to hardcode these columns in, because `print_table` is used in many other commands other than just `cl uls` and it may break backwards compatibility (and I'm not sure if all other tables will have time columns necessarily named "time")... what do you think @percyliang ?

Reply:
Would the try statement prevent backward incompatibility?